plenum/server/node.py

```diff
@@ -2122,10 +2122,6 @@ def unpackClientMsg(self, msg, frm):
                     self.discard(msg_dict,
                                  reason="view change in progress",
                                  logMethod=logger.debug)
-                    self.send_nack_to_client((idr_from_req_data(msg_dict),
```

Comment:
Please fix the corresponding tests:
plenum/test/view_change/test_client_req_during_view_change.py
plenum/test/view_change/test_client_req_during_view_change_integration.py

Reply:
Actually, it seems only `plenum/test/view_change/test_client_req_during_view_change_integration.py` needs to be changed